source/unified-test-format/unified-test-format.rst

```diff
@@ -2492,7 +2560,224 @@ An example of this operation follows::
+  is waiting to be observed on the provided client. The assertions laid out in
+  `expectedEvent`_ MUST be used to determine if an observed event matches
+  ``event``. ``event`` SHOULD have an event type that was included in the
+  ``client``'s ``observeEvents`` field.
+
+- ``count``: Required integer. The number of matching events to wait for before
+  resuming execution of subsequent operations.
+
+For example, the following instructs the test runner to wait for at least one
+poolClearedEvent to be published::
+
+    - name: waitForEvent
+      object: testRunner
+      arguments:
+        client: *client0
+        event:
+          poolClearedEvent: {}
+        count: 1
+
+
+assertEventCount
+~~~~~~~~~~~~~~~~
+
+The "assertEventCount" operation instructs the test runner to assert the
+specified MongoClient has published a specific, matching event a given number of
+times so far in the test. Note that this includes any events published before
+the waitForEvent operation started.
 
+The following arguments are supported:
+
+- ``client``: Required string. Client entity whose events the runner will be
+  counting.
+
+- ``event``: Required `expectedEvent`_ object. The event which the test runner
+  will be counting on the provided client. The assertions laid out in
+  `expectedEvent`_ MUST be used to determine if an observed event matches
+  ``event``. ``event`` SHOULD have an event type that was included in the
+  ``client``'s ``observeEvents`` field.
+
+- ``count``: Required integer. The exact number of matching events that
+  ``client`` should have seen so far.
+
+For example, the following instructs the test runner to assert that
+a single PoolClearedEvent was published::
+
+      - name: assertEventCount
+        object: testRunner
+        arguments:
+          client: *client0
+          event:
+            poolClearedEvent: {}
+          count: 1
+
+recordTopologyDescription
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The "recordTopologyDescription" operation instructs the test runner to retrieve
+the specified MongoClient's current `TopologyDescription <entity_topologydescription_>`_ and store it in
+the `Entity Map`_.
+
+The following arguments are suported:
+
+- ``client``: Required string. Client entity whose TopologyDescription will be recorded.
+
+- ``id``: Required string. The name with which the TopologyDescription will be
+  stored in the `Entity Map`_.
+
+For example::
+
+    - name: recordTopologyDescription
+      object: testRunner
+      arguments:
+        client: *client0
+        id: &postInsertTopology postInsertTopology
+
+assertTopologyType
+~~~~~~~~~~~~~~~~~~
+
+The ``assertTopologyType`` operation instructs the test runner to assert that
+the given `TopologyDescription <entity_topologydescription_>`_ has a particular TopologyType.
+
+The following arguments are supported:
+
+- ``topologyDescription``: Required string. See
+  `commonOptions_topologyDescription`_.
```

Comment:
This appears to be the only place that `topologyDescription` is accepted as an option. I'd propose removing it from "Common Options" and just defining it here.

Reply:
done